.cirrus.yml

```diff
@@ -42,5 +48,32 @@ release_task:
     - python3.7 -m ensurepip
     - python3.8 -m ensurepip
   build_script: ./make-nix-release.sh
+  script: |
+    #!/usr/bin/env bash
+
+    if [[ "$CIRRUS_RELEASE" == "" ]]; then
+      echo "Not a release. No need to deploy!"
+      exit 0
+    fi
+
+    if [[ "$GITHUB_TOKEN" == "" ]]; then
+      echo "Please provide GitHub access token via GITHUB_TOKEN environment variable!"
+      exit 1
+    fi
+
+    file_content_type="application/octet-stream"
+    files_to_upload=(releases/*)
+
+    for fpath in $files_to_upload
+    do
+      echo "Uploading $fpath..."
+      name=$(basename "$fpath")
+      url_to_upload="https://uploads.github.com/repos/$CIRRUS_REPO_FULL_NAME/releases/$CIRRUS_RELEASE/assets?name=$name"
```

Comment:
I think this will have to be `$CIRRUS_TAG` instead of `$CIRRUS_RELEASE`, if my comment above holds.